src/Coverage/Transformer.hs

```diff
@@ -17,11 +17,12 @@ import qualified Data.Map.Strict      as Map
 import           Elm.Utils            ((|>))
 import           Reporting.Annotation (Located (..))
 import           Reporting.Region
+import           System.FilePath
```

Comment:
maybe either qualified import or explicit exposing here would increase of chance of getting this merged - it seems more inline with code style used across the codebase.